common/src/main/java/com/wynntils/models/ingredients/IngredientProfilesModel.java

```diff
@@ -69,6 +73,16 @@ private void tryLoadIngredientList() {
         });
     }
 
+    private void tryLoadInvertedList() {
+        Download dl = Managers.Net.download(UrlId.DATA_ATHENA_ITEM_LIST);
+        dl.handleJsonObject(json -> {
+            Type hashmapType = new TypeToken<HashMap<String, String>>() {}.getType();
```

Comment:
Is this used?

Reply:
Yes, for ingredients. I could almost, but not completely, remove this. I think it was some additional metadata that was needed by the ingredients. See also my previous comment below for why ingredient looks like they have new code even though it doesn't.

Reply:
Github reviews.. I meant the variable `hashmapType`. I can't see any use for it?

Reply:
No, you are right. I copied old code and pruned unused parts, but missed this.